frontend/src/metabase/core/components/ButtonGroup/ButtonGroup.styled.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "@emotion/styled";
+import Button from "metabase/core/components/Button";
+
+export const ButtonGroupRoot = styled.div`
+  display: inline-block;
+
+  ${Button.Root} {
+    &:not(:last-child) {
+      border-right-width: 0.5px;
+      border-top-right-radius: 0;
+      border-bottom-right-radius: 0;
+    }
+
+    &:not(:first-child) {
+      border-left-width: 0.5px;
+      border-top-left-radius: 0;
+      border-bottom-left-radius: 0;
+    }
```

Comment:
Should we use `last-of-type` and `first-of-type` here? It's nothing serious, but emotion complains about them being potentially unsafe for SSR in the developer console. We don't use SSR, but the errors are pretty annoying. Maybe there is a way to disable these warnings, but I have no clue, unfortunately :(

Reply:
Fixed